backend/src/main/java/de/tobias/intestinalinspector/controller/PainController.java

```diff
@@ -49,8 +47,8 @@ public PainController(PainService painService) {
     }
 
     @PutMapping("/update/{id}")
```

Comment:
Same as before, address of put should be same as for a get by id, no namespacing with `update`